bin/annotate.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+
+import pandas as pd
+import scanpy as sc
+import scrublet as scr
+from arg_utils import parse_cl_paths
+
+
+def annotate(output_path: Path) -> str:
+    """
+    This function takes a filtered feature matrix file and annotates it,
+    saving the result in the input parameter output_path.
+
+    It annotates various gene types as well as doublet scoring by scrublet,
+    also calculating QC metrics while it's at it.
+    """
+    # Define glob patterns for each matrix type
```

Comment:
i don't want to hardcode the velocyto file output name like this but i can't think of a cleaner solution that also doesn't add a nextflow output from the velocyto process and an input into the annotate process